Testing/Unit/PowerShell/Orchestrator/Connect-Tenant.Tests.ps1

```diff
@@ -0,0 +1,98 @@
+$OrchestratorPath = '../../../../PowerShell/ScubaGear/Modules/Orchestrator.psm1'
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $OrchestratorPath) -Function 'Connect-Tenant' -Force
+
+InModuleScope Orchestrator {
+    Describe -Tag 'Orchestrator' -Name 'Invoke-Connection' {
+        Context 'When interactively connecting to commercial Endpoints' {
+            BeforeAll {
+                [Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSUseDeclaredVarsMoreThanAssignments', 'ConnectParams')]
+                $ConnectParams = @{
+                    LogIn           = $true
+                    M365Environment = "commercial"
+                    BoundParameters = @{}
+                }
+                function Connect-Tenant {}
+                Mock -ModuleName Orchestrator Connect-Tenant -MockWith {@()}
+            }
+            It 'With -ProductNames "aad", connects to Microsoft Graph' {
+                $ConnectParams += @{
+                    ProductNames = 'aad'
+                }
+                $FailedAuthList = Invoke-Connection @ConnectParams
+                $FailedAuthList.Length | Should -Be 0
```

Comment:
Based on the mocking approach i beeliev this is a tautology.  The mocked Connect-Tenant will always return an empty list.  So its length will always be true.

Reply:
Noted for change